src/dawn/Optimizer/PassManager.cpp

```diff
@@ -51,6 +51,8 @@ bool PassManager::runPassOnStecilInstantiation(
     return false;
   }
 
+  DAWN_ASSERT(instantiation->getIIR()->checkTreeConsistency());
```

Comment:
assert with message what pass failed it?

Reply:
done